usim/_primitives/timing.py

```diff
@@ -476,13 +481,19 @@ def __repr__(self):
 
     Using interval causes iteration to *resume at* regular times,
     even if the current activity is :term:`suspended <Suspension>`
-    in the loop body - the pause is shortened if necessary.
+    in the loop body - the pause is shortened as necessary.
+    This effectively creates a "clock" that ticks every ``period``
+    and runs the loop body.
+    If the loop body is suspended for longer than ``period`` so that a regular
+    cannot be met, :py:exc:`~.IntervalExceeded` is raised.
```

Comment:
A word is missing in this sentence. What do you think about *interval*?

```suggestion
    interval cannot be met, :py:exc:`~.IntervalExceeded` is raised.
```